x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_actions/get_user_action_stats.ts

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  CaseResponse,
+  CaseSeverity,
+  CaseStatuses,
+  ConnectorTypes,
+} from '@kbn/cases-plugin/common/api';
+
+import {
+  globalRead,
+  obsSec,
+  obsSecRead,
+  noKibanaPrivileges,
+  secOnly,
+  secOnlyRead,
+  superUser,
+} from '../../../../common/lib/authentication/users';
+import { getCaseUserActionStats } from '../../../../common/lib/user_actions';
+import { getPostCaseRequest, postCaseReq, postCommentUserReq } from '../../../../common/lib/mock';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import {
+  createCase,
+  createComment,
+  deleteAllCaseItems,
+  updateCase,
+  superUserSpace1Auth,
+} from '../../../../common/lib/utils';
+
+const CASE_UPDATE_DATA = (id: string, version: string) => ({
+  status: CaseStatuses.open,
+  severity: CaseSeverity.MEDIUM,
+  title: 'new title',
+  description: 'new desc',
+  settings: {
+    syncAlerts: false,
+  },
+  tags: ['one', 'two'],
+  assignees: [
+    {
+      uid: '123',
+    },
+  ],
+  connector: {
+    id: 'my-id',
+    name: 'Jira',
+    type: ConnectorTypes.jira as const,
+    fields: null,
+  },
+  id,
+  version,
+});
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
```

Comment:
Let's add a test in the trial folder (`x-pack/test/cases_api_integration/security_and_spaces/tests/trial/internal`) where we push a connector (connectors can be used only in trial)and ensure they are part of the `total_actions` and not part of `total_comments`.